src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AzureFunction/InvokeFunctionAsyncWrapper.cs

```diff
@@ -39,11 +39,15 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
             }
 
             var functionDetails = new FunctionDetails(functionContext, agent);
-            // TODO: add validation for FunctionDetails? 
+            if (!functionDetails.IsValid())
+            {
+                agent.Logger.Debug($"{WrapperName}: FunctionDetails are invalid, can't instrument this invocation.");
```

Comment:
Not a huge deal, but this could lead to a lot of logging if the customer's config is somehow messed up and every invocation fails validation. I wonder if we couldn't throw an exception here so that instrumentation will eventually get turned off it it happens too many times?

Reply:
I was unsure if we wanted to do that.  There are some other places we could do the same as well.  Let me put up a commit with something.

Reply:
Seems reasonable. I'm sure others will comment on the feature PR if they have a better idea...